docs/manual/upgrades.md

```diff
@@ -0,0 +1,129 @@
+Depending on the defined [Pod update strategy](../cluster_spec.md#podupdatestrategy) the operator will handle the recreation of the Pods differently.
+In the default setting `ReplaceTransactionSystem` the operator will recreate the storage Pods zone by zone and replaces Pods of the transaction system (log and stateless processes).
+The replacement of the transaction system has the benefit that the recoveries of the transaction system is reduced to potentially one recovery.
+Once all Pods are updated to the new image the upgrade is done and the cluster status of the FoundationDB cluster resource in Kubernetes should show that the reconciliation is done.
+
+### Known issues
+
+There are a number of known issues that can occur during an upgrade of FoundationDB running on Kubernetes.
+Most of the issues are only a challenged for version incompatible upgrades and most issues are handled automatically by the operator.
+If an issue cannot be handled by the operator we will outline a way to manually resolve the issue.
+
+#### Process restarted during staging Phase
+
+During the staging phase it can happen that a `fdbserver` process gets restarted by an external event like the OOM killer or any issue that results in the `fdbserver` crashing.
+In that case the `fdbmonitor` will restart the `fdbserver` process with the new version and for version incompatible upgrades, that means this process is not able to connect to the cluster.
+The operator will mark this Process Group with the `MissingProcesses` condition, since the process will be missing in the [cluster status json](https://apple.github.io/foundationdb/mr-status.html).
+If the process has this condition longer than the [IgnoreMissingProcessesSeconds](../cluster_spec.md#foundationdbclusterautomationoptions) setting it will be ignored in the [Restart Phase](#restart-phase) to ensure the operator can move forward with the upgrade.
+This also means that for a short time the cluster might be running with less resources.
+
+Processes that a restarted to early can be identified with the `MissingProcesses` condition.
+Using `kubectl` to fetch the cluster status and searching for all Process Groups with this condition gives an overview of those processes restarted to early.
+
+The operator will handle those cases automatically and there is no need for manual intervention as long as not too many processes are restarted from the outside.
+In that case the best way forward is to restart all processes still running on the old version, this can either be done manually or by using the [kubectl fdb plugin](../../kubectl-fdb/Readme.md) with the following command:
+
+```bash
+kubectl fdb restart -c cluster --process-condition=IncorrectCommandLine
+```
+
+or
+
+```bash
+kubectl fdb restart -c cluster --process-condition=MissingProcesses
```

Comment:
This doesn't help (because the MissingProcesses have already restarted), right?

Reply:
Correct, let me remove that.